src/utils/jira/getTicketComments.ts

```diff
@@ -0,0 +1,24 @@
+import axios from "axios";
```

Comment:
So we can axios in the end?

Reply:
Yes, it's been the standard for a while.
Thought process: about half the team has a hard time with the fetch API, pay bundle size for DX.